docs/metecho-how-to.md

```diff
@@ -0,0 +1,219 @@
+- Select the location to retrieve changes.
+  - Use the preselected Package Directory unless you know of a specific reason
+    to select a different option.
+- Select the changes to retrieve (or ignore changes you do not intend to
+  retrieve).
+- Create a commit message that briefly describes the changes.
+- Select Retrieve Selected Changes.
+- Retrieving changes can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the changes have been retrieved.
+- Note that the Developer, commit message, and date now appear in a Commit
+  History list in the Task detail view.
+
+## Step 7 - Submit Task Changes for Testing
+**Submit Changes** - *Document all the changes a Developer made in a Salesforce
+Dev Org so that a Tester can test the work and leave a review. This action
+creates a pull request in GitHub.*
+
+**Pull Request** - *A way to propose changes on GitHub so that the maintainers
+of a Project can review them and accept the changes or request revisions.*
+
+- When the Developer is finished working and retrieving changes, they need to
+  select Submit Task for Testing (button located just above the Developer card
+  in the Task detail view).
+- Each commit message briefly described what changed. Now, when submitting the
+  Task for testing, describe all the critical changes and why they were made so
+  the Tester will understand how to test them.
+- Select Submit Task for Testing.
+
+## Step 8 - Assign a Tester
+**Tester** - *The person who will look at the Developer’s work, and then approve
+the work or request changes that must be addressed before the Task can be
+completed.*
+
+- Assign yourself or another person to be the Tester for each Task you created.
+- Testers can be assigned at any time.
+- If you do not see the person you want to assign, contact an admin for the
+  repository on GitHub and ensure that the person is a collaborator. Then
+  re-sync GitHub collaborators in Metecho.
+- If you need to change or remove the Task Tester, use the drop down menu on the
+  Tester card in the Task detail view.
+
+## Step 9 - Create Test Org
+**Test Org** - *A temporary Salesforce org where the Tester can look over the
+Developer’s work on a specific Task. Test Orgs expire after 30 days.*
+
+- Remember, you can review the Next Steps for This Task list to see progress and
+  who is responsible for the next step.
+- To begin testing, a Task Tester will need to select Create Org in the Test Org
+  card.
+- Test Org creation can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the Test Org is ready.
+
+## Step 10 - Test Changes in Test Org
+- **This step takes place in the new Salesforce Test Org, not in Metecho.**
+- Select View Org on the Test Org card to navigate out of Metecho and test the
+  Developer’s changes in the temporary Salesforce Test Org.
+
+## Step 11 - Submit a Review
+- When the Tester is finished testing the Developer’s changes in the Test Org,
```

Comment:
Users will need to navigate back to Metecho to submit the review, right? If so, it might help to mention that since they had to navigate to Salesforce to test the changes in the Test Org